main.go

```diff
@@ -71,6 +71,7 @@ func main() {
 	shutdown := otelutils.TracerProviderFromEnv(tracerCtx, serviceName, func(e error) { log.Fatal(e) })
 	defer shutdown()
 
+	//nolint:staticcheck
```

Comment:
Encoded connections are deprecated in NATS. The solution is to handle marshaling and unmarshaling the JSON in this service, but I want to get this project out the door so that I can help with other tasks. For the time being, I disabled these lint checks, and created a [JIra issue](https://cyverse.atlassian.net/browse/CORE-2020) to address this.